### DISCUSSION TITLE
Welcome to erpsaas Discussions!

### DISCUSSION THREAD

**Discussion by andrewdwallo** [andrewdwallo/erpsaas#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nnq1** [comment]
I am testing the code and creating entities, until now it goes smoothly.

**Comment by alneamahaqlan** [comment]
why when i click on register
https://erpsaas.test/company/register
return not found

**Reply by LSANGARI** [reply]
Hello, were you able to solve the problem, if you did, can you please tell me how you solved it?

**Reply by andrewdwallo** [reply]
Can you provide anything useful for me to pinpoint the issue? Any logs, exceptions? What you provided for me essentially does nothing to help.

**Reply by LSANGARI** [reply]
Hi Andrew, nice talking to you...

What I did is clone the 2.x repo according to your instructions.

git clone https://github.com/andrewdwallo/erpsaas.git
cd erpsaas
composer install
npm install
cp .env.example .env
php artisan key:generate
php artisan migrate
npm run build
npm run dev

Once all the indicated steps have been completed, including the seeds, when you want to enter http://erpsaas.test/company/login or http://erpsaas.test/company/register I obtain

Not Found
The requested URL was not found on this server.

php artisan route:list

   GET|HEAD / .............................................. .................................................. .......
   POST _ignition/execute-solution ignition.executeSolution › Spatie\LaravelIgnition › ExecuteSolutionController
   GET|HEAD _ignition/health-check ............ ignition.healthCheck › Spatie\LaravelIgnition › HealthCheckController
   POST _ignition/update-config ......... ignition.updateConfig › Spatie\LaravelIgnition › UpdateConfigController
   GET|HEAD admin ....................................... filament.admin.pages.dashboard › Filament\Pages › Dashboard
   GET|HEAD admin/login .......................................... filament. admin.auth.login › Filament\Pages › Login
   POST admin/logout .............................. filament.admin.auth.logout › Filament\Http › LogoutController
   GET|HEAD api/user ........................................... .................................................. ..
   GET|HEAD company ........................... filament.company.tenant › Filament\Http › RedirectToTenantController
   GET|HEAD company/company-invitations/{invitation} filament.company.company-invitations.accept › Wallo\FilamentCom…
   GET|HEAD company/login ........................... filament.company.auth.login › Wallo\FilamentCompanies › Login
   POST company/logout .......................... filament.company.auth.logout › Filament\Http › LogoutController
   GET|HEAD company/new ............. filament.company.tenant.registration › App\Filament\Company\Pages\CreateCompany
   GET|HEAD company/oauth/{provider} filament.company.oauth.redirect › Wallo\FilamentCompanies › OAuthController@red…
   GET|HEAD company/oauth/{provider}/callback filament.company.oauth.callback › Wallo\FilamentCompanies › OAuthContr…
   GET|HEAD company/password-reset/request filament.company.auth.password-reset.request › Filament\Pages › RequestPa…
   GET|HEAD company/password-reset/reset filament.company.auth.password-reset.reset › Filament\Pages › ResetPassword
   GET|HEAD company/privacy-policy ........... filament.company.auth.policy › Wallo\FilamentCompanies › PrivacyPolicy
   GET|HEAD company/register .................... filament.company.auth.register › Wallo\FilamentCompanies › Register
   GET|HEAD company/terms-of-service ................... filament.company.auth.terms › Wallo\FilamentCompanies › Terms
   GET|HEAD company/{tenant} .......................... filament.company.pages.dashboard › Filament\Pages › Dashboard
   GET|HEAD company/{tenant}/banking/accounts filament.company.resources.banking.accounts.index › App\Filament\Compa…
   GET|HEAD company/{tenant}/banking/accounts/create filament.company.resources.banking.accounts.create › App\Filame…
   GET|HEAD company/{tenant}/banking/accounts/{record}/edit filament.company.resources.banking.accounts.edit › App\F…
   GET|HEAD company/{tenant}/hr/departments filament.company.resources.hr.departments.index › App\Filament\Company\R…
   GET|HEAD company/{tenant}/hr/departments/create filament.company.resources.hr.departments.create › App\Filament\C…
   GET|HEAD company/{tenant}/hr/departments/{record}/edit filament.company.resources.hr.departments.edit › App\Filam…
   GET|HEAD company/{tenant}/profile .... filament.company.tenant.profile › Wallo\FilamentCompanies › CompanySettings
   GET|HEAD company/{tenant}/settings/appearance filament.company.pages.settings.appearance › App\Filament\Company\P…
   GET|HEAD company/{tenant}/settings/categories filament.company.resources.settings.categories.index › App\Filament…
   GET|HEAD company/{tenant}/settings/categories/create filament.company.resources.settings.categories.create › App\…
   GET|HEAD company/{tenant}/settings/categories/{record}/edit filament.company.resources.settings.categories.edit …
   GET|HEAD company/{tenant}/settings/company-profile filament.company.pages.settings.company-profile › App\Filament…
   GET|HEAD company/{tenant}/settings/currencies filament.company.resources.settings.currencies.index › App\Filament…
   GET|HEAD company/{tenant}/settings/currencies/create filament.company.resources.settings.currencies.create › App\…
   GET|HEAD company/{tenant}/settings/currencies/{record}/edit filament.company.resources.settings.currencies.edit …
   GET|HEAD company/{tenant}/settings/default filament.company.pages.settings.default › App\Filament\Company\Pages\S…
   GET|HEAD company/{tenant}/settings/discount

If you need any type of complementary information, do not hesitate to request it.

Thank you very much for all your contributions

**Comment by andrewdwallo** [comment]
This sounds like a web server configuration error. Other than running the commands you showed, did you happen to correctly configure the `.env` file accordingly?

**Reply by LSANGARI** [reply]
The .env practically has the minimum config, and I checked it against the .env of the erpsaas-1.x version and except for the logical changes everything is in order :(

APP_NAME=erpsaas
APP_ENV=local
APP_KEY=base64:q5+ksBdNBc29yPJZEY3cQmny5UAjDSHCYTKzlmrz1IA=
APP_DEBUG=true
APP_URL=http://erpsaas.test/

LOG_CHANNEL=stack
LOG_DEPRECATIONS_CHANNEL=null
LOG_LEVEL=debug

DB_CONNECTION=mysql
DB_HOST=127.0.0.1
DB_PORT=3306
DB_DATABASE=laravel
DB_USERNAME=root
DB_PASSWORD=

BROADCAST_DRIVER=log
CACHE_DRIVER=file
FILESYSTEM_DISK=local
QUEUE_CONNECTION=sync
SESSION_DRIVER=database
SESSION_LIFETIME=120

MEMCACHED_HOST=127.0.0.1

REDIS_HOST=127.0.0.1
REDIS_PASSWORD=null
REDIS_PORT=6379

MAIL_MAILER=smtp
MAIL_HOST=mailpit
MAIL_PORT=1025
MAIL_USERNAME=null
MAIL_PASSWORD=null
MAIL_ENCRYPTION=null
MAIL_FROM_ADDRESS="hello@example.com"
MAIL_FROM_NAME="${APP_NAME}"

AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1
AWS_BUCKET=
AWS_USE_PATH_STYLE_ENDPOINT=false

PUSHER_APP_ID=
PUSHER_APP_KEY=
PUSHER_APP_SECRET=
PUSHER_HOST=
PUSHER_PORT=443
PUSHER_SCHEME=https
PUSHER_APP_CLUSTER=mt1

VITE_APP_NAME="${APP_NAME}"
VITE_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
VITE_PUSHER_HOST="${PUSHER_HOST}"
VITE_PUSHER_PORT="${PUSHER_PORT}"
VITE_PUSHER_SCHEME="${PUSHER_SCHEME}"
VITE_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"

**Reply by andrewdwallo** [reply]
Did you configure your database correctly? It shows that the database name is "laravel", but usually the database name would match the `APP_NAME`, which would be "erpsaas".

One more thing.. change `APP_URL` to "http://erpsaas.test" without the leading backslash. That is probably what is causing your issue.

**Reply by alneamahaqlan** [reply]
I try  delete leading backslash but not work I change server to nginx in laragon and it's work

**Reply by andrewdwallo** [reply]
Oh, okay that makes sense. I recall the time I transitioned from Windows with Laragon to using Nginx in place of Apache. Although I'm uncertain of the precise reasons, I believe it involved a compatibility issue between Laravel and Apache.

**Reply by LSANGARI** [reply]
Hello, yes I am using Laragon with Apache :( but I couldn't get it to work, now according to that comment I am going to use xampp in my manjaro and then I will tell you if I get involved

**Reply by andrewdwallo** [reply]
Laragon with Nginx should be just fine. As long as you aren't using Apache. The operating system being used shouldn't matter in this case.

**Reply by LSANGARI** [reply]
They were right, the problem is the Apache, now I will look for what the problem is. :)

**Reply by andrewdwallo** [reply]
Okay sounds good. If you find the actual cause of the issue, please let me know because I wasn't able to find it in the past. 👍

**Reply by LSANGARI** [reply]
SOLVED!!!!!!!! :) I was able to run version 2 in Apache instead of Nginx, the problem lies in the way Nginx and Apache work.

As you know, the rewriter module is the problem and that's where the NOT FOUND error for the urls comes from.

http://erpsaas.test/company/login http://erpsaas.test/company/register

because of how the Apache performs redactions internally.

So that everything works without problems in Nginx, if you are using Windows using Laragon, you can edit the following file.

{LARAGON_ROOT}\usr\tpl\VirtualHost.tpl

Adding the following inside the Directory tag


```
<IfModule mod_rewrite.c>
<IfModule mod_negotiation.c>
      Options -MultiViews -Indexes
  </IfModule>

     RewriteEngine On

     #HandleAuthorizationHeader
     RewriteCond %{HTTP:Authorization} .
     RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]

     # Redirect Trailing Slashes If Not A Folder...
     RewriteCond %{REQUEST_FILENAME} !-d
     RewriteCond %{REQUEST_URI} (.+)/$
     RewriteRule ^ %1 [L,R=301]

     # Send Requests To Front Controller...
     RewriteCond %{REQUEST_FILENAME} !-d
     RewriteCond %{REQUEST_FILENAME} !-f
     RewriteRule ^ index.php [L]
</IfModule>
```


Since if you don't do this, Laragon reestablishes the vhosts that are called auto.xxxx.test.conf at each startup.

Option 2

Simply define your vhost with the values above indicating y in the file name xxxx.test.conf without the auto prefix.

Option 3

Edit the file that is in your project path / public

and modify the .htaccess file with the values indicated above

This is the easiest option since you don't have to touch anything of the Laragon.

**Reply by andrewdwallo** [reply]
Nice! Thank you for taking the time to find the issue and sharing this detailed solution! For anyone else experiencing similar issues, this solution might be the key to resolving them. I encourage you to give it a try and report back your results. Thanks again!

**Comment by andrewdwallo** [comment]
I don't know what to tell you. It seems like a server configuration issue. Try running `php artisan optimize:clear` and see if that does anything. If all else fails, you can try reinstalling.

**Reply by LSANGARI** [reply]
Thanks for your time Andrew but I give up, I know it will be wrong but version 2.x doesn't work for me. Version 1.x is brilliant, very good what you have achieved.

**Reply by andrewdwallo** [reply]
Thanks! Sorry you couldn't get it to work. The new version is a lot better in my opinion. Plus, there will be a new release soon with a lot of cool features..

**Comment by demeus** [comment]
Hi! I'm the only one with an error when running? 
`php artisan db:seed`

 Illuminate\Database\QueryException 

  SQLSTATE[HY000]: General error: 1 no such table: states (Connection: , SQL: select * from "states" where "country_id" = MW order by RANDOM() limit 1)
It looks like there is no migration to create the states table

**Comment by andrewdwallo** [comment]
@demeus Try running `php artisan optimize:clear`. Then use `npm run dev` and while your console is running use `php artisan db:seed`.

**Reply by vachzar** [reply]
> @demeus Try running `php artisan optimize:clear`. Then use `npm run dev` and while your console is running use `php artisan db:seed`.

Hi, I still got error while do `php artisan db:seed`
"SQLSTATE[HY000]: General error: 1 no such table: states (Connection: , SQL: select * from "states" where "country_id" = US order by RANDOM() limit 1)"

**Reply by andrewdwallo** [reply]
https://github.com/andrewdwallo/erpsaas/issues/93#issuecomment-2591545596

**Comment by isach5** [comment]
I'm bumping into an issue after login error: The POST method is not supported for route company/login.

**Reply by andrewdwallo** [reply]
Sorry for the late reply. Still having the issue?

**Reply by KhawarMehfooz** [reply]
Hi,

I am also facing the same issue for both login and register.
```php
The POST method is not supported for route company/login. Supported methods: GET, HEAD.
```

```php
The POST method is not supported for route company/register. Supported methods: GET, HEAD.
```

### Routes List

```bash
php artisan route:list

  GET|HEAD   / .......................................................................................................................................... 
  POST       _ignition/execute-solution ................................... ignition.executeSolution › Spatie\LaravelIgnition › ExecuteSolutionController
  GET|HEAD   _ignition/health-check ............................................... ignition.healthCheck › Spatie\LaravelIgnition › HealthCheckController
  POST       _ignition/update-config ............................................ ignition.updateConfig › Spatie\LaravelIgnition › UpdateConfigController
  GET|HEAD   admin .......................................................................... filament.admin.pages.dashboard › Filament\Pages › Dashboard
  POST       admin/logout ................................................................. filament.admin.auth.logout › Filament\Http › LogoutController
  POST       api/plaid/webhook ..................................................................................... PlaidWebhookController@handleWebhook
  GET|HEAD   api/user ................................................................................................................................... 
  GET|HEAD   company ............................................................... filament.company.tenant › Filament\Http › RedirectToTenantController
  GET|HEAD   company/invitations/{invitation} ........ filament.company.invitations.accept › Wallo\FilamentCompanies › CompanyInvitationController@accept
  GET|HEAD   company/login ................................................................ filament.company.auth.login › Wallo\FilamentCompanies › Login
  POST       company/logout ............................................................. filament.company.auth.logout › Filament\Http › LogoutController
  GET|HEAD   company/new ................................................ filament.company.tenant.registration › App\Filament\Company\Pages\CreateCompany
  GET|HEAD   company/oauth/{provider} .................... filament.company.oauth.redirect › Wallo\FilamentCompanies › OAuthController@redirectToProvider
  GET|HEAD   company/oauth/{provider}/callback ....... filament.company.oauth.callback › Wallo\FilamentCompanies › OAuthController@handleProviderCallback
  GET|HEAD   company/password-reset/request ........................ filament.company.auth.password-reset.request › Filament\Pages › RequestPasswordReset
  GET|HEAD   company/password-reset/reset ................................... filament.company.auth.password-reset.reset › Filament\Pages › ResetPassword
  GET|HEAD   company/privacy-policy .............................................. filament.company.auth.policy › Wallo\FilamentCompanies › PrivacyPolicy
  GET|HEAD   company/register ....................................................... filament.company.auth.register › Wallo\FilamentCompanies › Register
  GET|HEAD   company/terms-of-service ..................................................... filament.company.auth.terms › Wallo\FilamentCompanies › Terms
  GET|HEAD   company/{tenant} ............................................................. filament.company.pages.dashboard › Filament\Pages › Dashboard
  GET|HEAD   company/{tenant}/accounting/chart ............. filament.company.pages.accounting.chart › App\Filament\Company\Pages\Accounting\AccountChart
  GET|HEAD   company/{tenant}/banking/accounts filament.company.resources.banking.accounts.index › App\Filament\Company\Resources\Banking\AccountResourc…
  GET|HEAD   company/{tenant}/banking/accounts/create filament.company.resources.banking.accounts.create › App\Filament\Company\Resources\Banking\Accoun…
  GET|HEAD   company/{tenant}/banking/accounts/{record}/edit filament.company.resources.banking.accounts.edit › App\Filament\Company\Resources\Banking\A…
  GET|HEAD   company/{tenant}/common/offerings filament.company.resources.common.offerings.index › App\Filament\Company\Resources\Common\OfferingResourc…
  GET|HEAD   company/{tenant}/common/offerings/create filament.company.resources.common.offerings.create › App\Filament\Company\Resources\Common\Offerin…
  GET|HEAD   company/{tenant}/common/offerings/{record}/edit filament.company.resources.common.offerings.edit › App\Filament\Company\Resources\Common\Of…
  GET|HEAD   company/{tenant}/hr/departments filament.company.resources.hr.departments.index › App\Filament\Company\Resources\Core\DepartmentResource\Pa…
  GET|HEAD   company/{tenant}/hr/departments/create filament.company.resources.hr.departments.create › App\Filament\Company\Resources\Core\DepartmentRes…
  GET|HEAD   company/{tenant}/hr/departments/{record}/edit filament.company.resources.hr.departments.edit › App\Filament\Company\Resources\Core\Departme…
  GET|HEAD   company/{tenant}/manage-company ................................. filament.company.tenant.profile › App\Filament\Company\Pages\ManageCompany
  GET|HEAD   company/{tenant}/purchases/bills filament.company.resources.purchases.bills.index › App\Filament\Company\Resources\Purchases\BillResource\P…
  GET|HEAD   company/{tenant}/purchases/bills/create filament.company.resources.purchases.bills.create › App\Filament\Company\Resources\Purchases\BillRe…
  GET|HEAD   company/{tenant}/purchases/bills/{record} filament.company.resources.purchases.bills.view › App\Filament\Company\Resources\Purchases\BillRe…
  GET|HEAD   company/{tenant}/purchases/bills/{record}/edit filament.company.resources.purchases.bills.edit › App\Filament\Company\Resources\Purchases\B…
  GET|HEAD   company/{tenant}/purchases/vendors filament.company.resources.purchases.vendors.index › App\Filament\Company\Resources\Purchases\VendorReso…
  GET|HEAD   company/{tenant}/purchases/vendors/create filament.company.resources.purchases.vendors.create › App\Filament\Company\Resources\Purchases\Ve…
  GET|HEAD   company/{tenant}/purchases/vendors/{record} filament.company.resources.purchases.vendors.view › App\Filament\Company\Resources\Purchases\Ve…
  GET|HEAD   company/{tenant}/purchases/vendors/{record}/edit filament.company.resources.purchases.vendors.edit › App\Filament\Company\Resources\Purchas…
  GET|HEAD   company/{tenant}/reports ............................................... filament.company.pages.reports › App\Filament\Company\Pages\Reports
  GET|HEAD   company/{tenant}/reports/account-balances filament.company.pages.reports.account-balances › App\Filament\Company\Pages\Reports\AccountBalan…
  GET|HEAD   company/{tenant}/reports/account-transactions filament.company.pages.reports.account-transactions › App\Filament\Company\Pages\Reports\Acco…
  GET|HEAD   company/{tenant}/reports/accounts-payable-aging filament.company.pages.reports.accounts-payable-aging › App\Filament\Company\Pages\Reports\…
  GET|HEAD   company/{tenant}/reports/accounts-receivable-aging filament.company.pages.reports.accounts-receivable-aging › App\Filament\Company\Pages\Re…
  GET|HEAD   company/{tenant}/reports/balance-sheet ...... filament.company.pages.reports.balance-sheet › App\Filament\Company\Pages\Reports\BalanceSheet
  GET|HEAD   company/{tenant}/reports/cash-flow-statement filament.company.pages.reports.cash-flow-statement › App\Filament\Company\Pages\Reports\CashFl…
  GET|HEAD   company/{tenant}/reports/client-balance-summary filament.company.pages.reports.client-balance-summary › App\Filament\Company\Pages\Reports\…
  GET|HEAD   company/{tenant}/reports/client-payment-performance filament.company.pages.reports.client-payment-performance › App\Filament\Company\Pages\…
  GET|HEAD   company/{tenant}/reports/income-statement filament.company.pages.reports.income-statement › App\Filament\Company\Pages\Reports\IncomeStatem…
  GET|HEAD   company/{tenant}/reports/trial-balance ...... filament.company.pages.reports.trial-balance › App\Filament\Company\Pages\Reports\TrialBalance
  GET|HEAD   company/{tenant}/reports/vendor-balance-summary filament.company.pages.reports.vendor-balance-summary › App\Filament\Company\Pages\Reports\…
  GET|HEAD   company/{tenant}/reports/vendor-payment-performance filament.company.pages.reports.vendor-payment-performance › App\Filament\Company\Pages\…
  GET|HEAD   company/{tenant}/sales/clients filament.company.resources.sales.clients.index › App\Filament\Company\Resources\Sales\ClientResource\Pages\L…
  GET|HEAD   company/{tenant}/sales/clients/create filament.company.resources.sales.clients.create › App\Filament\Company\Resources\Sales\ClientResource…
  GET|HEAD   company/{tenant}/sales/clients/{record} filament.company.resources.sales.clients.view › App\Filament\Company\Resources\Sales\ClientResource…
  GET|HEAD   company/{tenant}/sales/clients/{record}/edit filament.company.resources.sales.clients.edit › App\Filament\Company\Resources\Sales\ClientRes…
  GET|HEAD   company/{tenant}/sales/estimates filament.company.resources.sales.estimates.index › App\Filament\Company\Resources\Sales\EstimateResource\P…
  GET|HEAD   company/{tenant}/sales/estimates/create filament.company.resources.sales.estimates.create › App\Filament\Company\Resources\Sales\EstimateRe…
  GET|HEAD   company/{tenant}/sales/estimates/{record} filament.company.resources.sales.estimates.view › App\Filament\Company\Resources\Sales\EstimateRe…
  GET|HEAD   company/{tenant}/sales/estimates/{record}/edit filament.company.resources.sales.estimates.edit › App\Filament\Company\Resources\Sales\Estim…
  GET|HEAD   company/{tenant}/sales/invoices filament.company.resources.sales.invoices.index › App\Filament\Company\Resources\Sales\InvoiceResource\Page…
  GET|HEAD   company/{tenant}/sales/invoices/create filament.company.resources.sales.invoices.create › App\Filament\Company\Resources\Sales\InvoiceResou…
  GET|HEAD   company/{tenant}/sales/invoices/{record} filament.company.resources.sales.invoices.view › App\Filament\Company\Resources\Sales\InvoiceResou…
  GET|HEAD   company/{tenant}/sales/invoices/{record}/edit filament.company.resources.sales.invoices.edit › App\Filament\Company\Resources\Sales\Invoice…
  GET|HEAD   company/{tenant}/sales/recurring-invoices filament.company.resources.sales.recurring-invoices.index › App\Filament\Company\Resources\Sales\…
  GET|HEAD   company/{tenant}/sales/recurring-invoices/create filament.company.resources.sales.recurring-invoices.create › App\Filament\Company\Resource…
  GET|HEAD   company/{tenant}/sales/recurring-invoices/{record} filament.company.resources.sales.recurring-invoices.view › App\Filament\Company\Resource…
  GET|HEAD   company/{tenant}/sales/recurring-invoices/{record}/edit filament.company.resources.sales.recurring-invoices.edit › App\Filament\Company\Res…
  GET|HEAD   company/{tenant}/services/connected-accounts filament.company.pages.services.connected-accounts › App\Filament\Company\Pages\Service\Connec…
  GET|HEAD   company/{tenant}/services/live-currency .... filament.company.pages.services.live-currency › App\Filament\Company\Pages\Service\LiveCurrency
  GET|HEAD   company/{tenant}/settings ............................................... filament.company.settings › App\Filament\Company\Clusters\Settings
  GET|HEAD   company/{tenant}/settings/adjustments filament.company.settings.resources.adjustments.index › App\Filament\Company\Clusters\Settings\Resour…
  GET|HEAD   company/{tenant}/settings/adjustments/create filament.company.settings.resources.adjustments.create › App\Filament\Company\Clusters\Setting…
  GET|HEAD   company/{tenant}/settings/adjustments/{record}/edit filament.company.settings.resources.adjustments.edit › App\Filament\Company\Clusters\Se…
  GET|HEAD   company/{tenant}/settings/appearance .. filament.company.settings.pages.appearance › App\Filament\Company\Clusters\Settings\Pages\Appearance
  GET|HEAD   company/{tenant}/settings/company-default filament.company.settings.pages.company-default › App\Filament\Company\Clusters\Settings\Pages\Co…
  GET|HEAD   company/{tenant}/settings/company-profile filament.company.settings.pages.company-profile › App\Filament\Company\Clusters\Settings\Pages\Co…
  GET|HEAD   company/{tenant}/settings/currencies filament.company.settings.resources.currencies.index › App\Filament\Company\Clusters\Settings\Resource…
  GET|HEAD   company/{tenant}/settings/currencies/create filament.company.settings.resources.currencies.create › App\Filament\Company\Clusters\Settings\…
  GET|HEAD   company/{tenant}/settings/currencies/{record}/edit filament.company.settings.resources.currencies.edit › App\Filament\Company\Clusters\Sett…
  GET|HEAD   company/{tenant}/settings/invoice ........... filament.company.settings.pages.invoice › App\Filament\Company\Clusters\Settings\Pages\Invoice
  GET|HEAD   company/{tenant}/settings/localization filament.company.settings.pages.localization › App\Filament\Company\Clusters\Settings\Pages\Localiza…
  GET|HEAD   company/{tenant}/transactions ..................... filament.company.pages.transactions › App\Filament\Company\Pages\Accounting\Transactions
  GET|HEAD   filament/exports/{export}/download ........................................... filament.exports.download › Filament\Actions › DownloadExport
  GET|HEAD   filament/imports/{import}/failed-rows/download ......... filament.imports.failed-rows.download › Filament\Actions › DownloadImportFailureCsv
  GET|HEAD   livewire/livewire.js ........................................................... Livewire\Mechanisms › FrontendAssets@returnJavaScriptAsFile
  GET|HEAD   livewire/livewire.min.js.map ..................................................................... Livewire\Mechanisms › FrontendAssets@maps
  GET|HEAD   livewire/preview-file/{filename} .................................. livewire.preview-file › Livewire\Features › FilePreviewController@handle
  POST       livewire/update ........................................................ livewire.update › Livewire\Mechanisms › HandleRequests@handleUpdate
  POST       livewire/upload-file ................................................ livewire.upload-file › Livewire\Features › FileUploadController@handle
  GET|HEAD   sanctum/csrf-cookie ...................................................... sanctum.csrf-cookie › Laravel\Sanctum › CsrfCookieController@show
  GET|HEAD   up ......................................................................................................................................... 
  GET|HEAD   user ......................................................................... filament.user.home › Filament\Http › RedirectToHomeController
  GET|HEAD   user/account .................................................................... filament.user.account › App\Filament\User\Clusters\Account
  GET|HEAD   user/account/personal-access-tokens filament.user.account.pages.personal-access-tokens › App\Filament\User\Clusters\Account\Pages\PersonalA…
  GET|HEAD   user/account/profile ................................ filament.user.account.pages.profile › App\Filament\User\Clusters\Account\Pages\Profile
  POST       user/logout ................................................................... filament.user.auth.logout › Filament\Http › LogoutController
  GET|HEAD   user/personal-access-tokens .................... filament.user.pages.personal-access-tokens › Wallo\FilamentCompanies › PersonalAccessTokens
  GET|HEAD   user/profile ............................................................... filament.user.pages.profile › Wallo\FilamentCompanies › Profile

                                                                                                                                     Showing [100] routes


```

**Reply by andrewdwallo** [reply]
@KhawarMehfooz Maybe this would help https://github.com/filamentphp/filament/discussions/11481#discussioncomment-8861715

**Reply by KhawarMehfooz** [reply]
@andrewdwallo Thanks, It worked.
